setup.py

```diff
@@ -55,14 +55,22 @@
         "pyarrow",
         "tldextract",
         "diskcache",
```

Comment:
This "diskcache" line can be deleted now (my bad for not noticing it was commented out lower in the file in my previous PR).

Reply:
ah I see it's duplicate!